security_mvp/Makefile

```
@@ -0,0 +1,13 @@
+.PHONY: install lint test run
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)